wandb/sdk/interface/interface.py

```diff
@@ -604,6 +604,26 @@ def publish_output(self, name: str, data: str) -> None:
     def _publish_output(self, outdata: pb.OutputRecord) -> None:
         raise NotImplementedError
 
+    def publish_output_raw(self, name: str, data: str) -> None:
+        # from vendor.protobuf import google3.protobuf.timestamp
+        # ts = timestamp.Timestamp()
+        # ts.GetCurrentTime()
+        # now = datetime.now()
+        if name == "stdout":
+            otype = pb.OutputRawRecord.OutputType.STDOUT
+        elif name == "stderr":
+            otype = pb.OutputRawRecord.OutputType.STDERR
+        else:
+            # TODO(jhr): throw error?
```

Comment:
I think it might be useful to either log it or throw erroe

Reply:
this is the same as publish output. was keeping it the same for this PR